packaging/leapp-el7toel8-deps.spec

```diff
@@ -18,11 +18,12 @@ URL:        https://leapp-to.github.io
 ##################################################
 %package -n %{lrdname}
 Summary:    Meta-package with system dependencies for leapp repository
-Provides:   leapp-repository-dependencies = 2
+Provides:   leapp-repository-dependencies = 3
```

Comment:
You correctly updated this capability, however, you need to update this one in the 2nd spec file as well. See `leapp-repository.spec`, line 33